File: Linguagem C/Lista_While/ex_007.c
//7) Faça um programa que leia um inteiro positivo N e mostre quais são os seus divisores. Por exemplo, os
//divisores de 30 são 1, 2, 3, 5, 6, 10, 15 e 30. Não deixe o usuário inserir um valor negativo. Imprima os
//números em uma única linha, separados entre si por um único espaço
#include <stdio.h>
int main (){
int n, i=1;
    do{
        printf("Digite um numero positivo: ");
        scanf("%d", &n);
    } while(n < 0 );

    while(i <= n){
       if(n % i == 0){
        
        printf("%d ", i);
       }
        
    i++;
    }
return 0;
}

File: Python/005_Lista_While/Ex_007.py
"""
7) Faça um programa que leia um inteiro positivo N e mostre quais são os seus divisores. Por exemplo, os 
divisores  de  30  são  1,  2,  3,  5,  6,  10,  15  e  30.  Não  deixe  o  usuário  inserir  um  valor  negativo.  Imprima  os 
números em uma única linha, separados entre si por um único espaço.
"""
n1 = int(input("Digite um numero para n1: 1"))

for num in range(n1, 0, -1):
    if n1 % num == 0:
        print(num, end=" ")
